pkg/middleware/quota.go

```diff
@@ -14,15 +14,15 @@ func Quota(quotaService quota.Service) func(string) web.Handler {
 		panic("quotaService is nil")
 	}
 	//https://open.spotify.com/track/7bZSoBEAEEUsGEuLOf94Jm?si=T1Tdju5qRSmmR0zph_6RBw fuuuuunky
```

Comment:
```
	//https://open.spotify.com/track/7bZSoBEAEEUsGEuLOf94Jm?si=T1Tdju5qRSmmR0zph_6RBw fuuuuunky
```
what is this line? 😆

Reply:
it's funky! no idea; it's there for a long time!

Reply:
@bergquist :smile: :egg: